mxcube3/ui/actions/general.js

```diff
@@ -165,6 +165,38 @@ export function getInitialState() {
         'Content-type': 'application/json'
       }
     });
+    const loadedSample = fetch('mxcube/api/v0.1/sample_changer/loaded_sample', {
```

Comment:
Question about the sample changer related routes; it seems like most of these are used only to initialize the sample changer, after for instance refresh. The routes could perhaps be grouped into one route with an appropriate name, in order to minimize requests and to simplify the code somewhat, we could for instance group:

   'mxcube/api/v0.1/sample_changer/contents'
   'mxcube/api/v0.1/sample_changer/loaded_sample'
   'mxcube/api/v0.1/sample_changer/state'

Finally this route have, if we have understood things correctly, a somewhat confusing name. We understand it to be the state for maintenance ?, if its the case why not include maintenance in the route name ?

   'mxcube/api/v0.1/sample_changer/get_global_state'